src/main/java/io/sealights/plugins/sealightsjenkins/utils/FileUtils.java

```diff
@@ -113,7 +113,7 @@ public static void tryDeleteFile(Logger logger, String filename) throws IOExcept
         logger.info("Try deleting file: " + filename);
         FilePath fpOnMaster = new FilePath(new File(filename));
         if (fpOnMaster.exists()) {
-            fpOnMaster.delete();
+            fpOnMaster.deleteRecursive();
```

Comment:
What happens if that's a file? Is it working? 
In addition, according to the log file you are deleting a file, but here you may delete a folder recursively. Please verify that's make sense and update the log accordingly.

Reply:
```deleteRecursive``` delete also single files, I checked and verified it in the source code.
In Java, a folder is also considered as a file, so to strict with single log message for both, i found the term 'file' good enough